azure-pipelines.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     displayName: Prepare conda
   - script: |
       source activate uarray
-      pip install git+https://github.com/Quansight-Labs/uarray.git
-      pip install git+https://github.com/Quansight-Labs/unumpy.git
```

Comment:
This should really not be commented. `udiff` should work with `master` of `uarray` and `unumpy`.

Reply:
OK. I will recover it in the next commit.

Reply:
Ouch. Let's remove the tests for now.